iot-hub/Samples/service/AutomaticDeviceManagementSample/Parameters.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using System;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments.
+    /// If the parameter is not supplied via command line args, it will look for it in environment variables.
+    /// </summary>
+    /// <remarks>
+    /// To get the connection string, log into https://azure.portal.com, go to Resources, open the IoT hub, open Shared Access Policies, open iothubowner, and copy a connection string.
+    /// </remarks>
+    internal class Parameters
+    {
+        [Option(
+            'c',
```

Comment:
Where did we land on this alias and constancy across the other samples?

Reply:
Looks like we're choosing -c consistently in this PR. We'll have other samples in this repo that need to be updated, right?

I'm all for updating it, but let's act globally.